framework/include/base/Assembly.h

```diff
@@ -1809,6 +1819,15 @@ class Assembly
   /// A pointer to const qrule_msm
   const QBase * _const_qrule_msm;
 
+private:
+  /// quadrature rule used on lower dimensional elements. This should always be
+  /// the same as the face qrule
+  const QBase * _const_current_qrule_lower;
+  /// quadrature rule used on lower dimensional elements. This should always be
+  /// the same as the face qrule
+  QBase * _current_qrule_lower;
```

Comment:
Do we have two variables that point to the same thing? The comments are the same.

Reply:
Oh I guess I should add doxygen that explains why I need both like the explanation I gave in https://github.com/idaholab/moose/pull/15259#discussion_r442375074

Reply:
@recuero I will add this doxygen and also do the method renaming. You don't have to do anything right now

Reply:
I added comments explaining why we have const and non-const versions